sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -141,7 +141,8 @@ export function browserConfig({ test = false, production = false } = {}) {
 
       nodeResolve({
         mainFields: ["module", "browser"],
-        preferBuiltins: false
+        preferBuiltins: false,
+        dedupe: ["buffer"]
```

Comment:
Please log an issue to remove this in the atom api epic so that we dont ship this accidentally

Reply:
Filed https://github.com/Azure/azure-sdk-for-js/issues/5983